source/frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -86,7 +86,6 @@ const getQueryParams = (filter: IPropertyFilter): IGeoSearchParams => {
     STREET_ADDRESS_1: filter.address,
     latitude: filter.latitude,
     longitude: filter.longitude,
-    forceExactMatch: true,
```

Comment:
forcing an exact match for map searches is not desired as per psp-6152. For example, searching for a pid with 01 in it should return all pids that contain 01. Using exact match only a pid of 000-000-001 or a pin of 1 would match.